config/crd/bases/mcad.ibm.com_queuejobs.yaml

```diff
@@ -43,6 +43,18 @@ spec:
                 properties:
                   minAvailable:
                     type: integer
+                  # dispatchdurationinseconds:
+                  #   type: integer
+                  dispatchDuration:
+                    description: wall clock duration time of appwrapper
+                    properties:
+                      expected:
```

Comment:
I don't see expected used anywhere in the `queuejob_controller_ex.go` so should remove

Reply:
go struct has the field which will be used in multi-cluster scenario hence we have to keep this in the schema. this filed will always be ignored as it is optional for now